pages/integrations/output_framework.rst

```diff
@@ -57,29 +66,36 @@ a processing pipeline which will be executed on each message coming from the sou
 not wish to forward.  It can also be used to add data to modify the contents of the outgoing
 message or to enrich it with additional data.
 
+Outbound Transports
+^^^^^^^^^^^^^^^^^^^
+
+Outbound Transport is the configuration of how the message is sent over the wire:
+
+- ``Enterprise STDOUT``
+    - Formatted messages will be displayed on the system's console.  This is included primarily as a debugging tool for pipeline changes.
+- ``Enterprise TCP Raw/Plaintext``
+    - Formatted messages will be sent as UTF-8 encoded plain text to the configured TCP endpoint (IP address and port). 
+    - :doc:`Output details<output_framework/output_tcp_raw>`
```

Comment:
personal would include the complete name of the output here. This will allow people using a screenreader to identify easier where this is going to PLUS you can search quicker `via Strg+f`